### ISSUE TITLE
Clean up labelling

### ISSUE THREAD

**Issue by samvanstroud** [umami-hep/atlas-ftag-tools#88]
- [ ] use labelling tool in resampling to generate training labels https://github.com/umami-hep/atlas-ftag-tools/pull/87
- [ ] rename flavour -> labels everywhere
- [ ] add in standard track labels schemes, replacing https://gitlab.cern.ch/atlas-flavor-tagging-tools/algorithms/salt/-/blob/main/salt/utils/class_names.py

**Comment by samvanstroud** [comment]
see https://github.com/umami-hep/atlas-ftag-tools/pull/94

**Comment by ecritell** [comment]
hi @samvanstroud happy to pick this up as part of my ongoing work on labelling

**Comment by samvanstroud** [comment]
Thanks @ecritell that would be awesome! I already got started in #103, maybe you could already take a look at that PR to get started?

**Comment by ecritell** [comment]
yes happy to do that, I see code coverage in https://github.com/umami-hep/atlas-ftag-tools/pull/103 has reduced hence CI failing, I can checkout your branch and add tests for the new lines if that helps. Other than that the PR looks good to me, unless you had in mind to add something else

**Comment by samvanstroud** [comment]
Thanks. I just merged it since the uncovered lines were existing I think. If you want to add coverage in a future MR that would be great, but no stress.
